samples/asgardeo-choreo-react-express/apps/server/package.json

```
@@ -5,10 +5,10 @@
   "type": "module",
   "author": "Yathindra Kodithuwakku <yathindrarawya123@gmail.com>",
   "scripts": {
-    "start": "yarn run server:local",
-    "prestart": "yarn run build",
-    "build": "rimraf dist && npx tsc",
-    "server:local": "concurrently \"npx tsc --watch\" \"NODE_OPTIONS='--experimental-specifier-resolution=node' nodemon -q dist/index.js\""
+    "start": "pnpm run server:local",
+    "prestart": "pnpm run build",
```

we may not need `run`. Please check

```suggestion
    "start": "pnpm server:local",
    "prestart": "pnpm build",
```

---

It works, But is it recommended?